src/components/Home.js

```diff
@@ -34,7 +33,7 @@ function Home(props) {
   const location = useLocation();
 
   const {data: timeseries} = useStickySWR(
-    'https://api.covid19india.org/v3/min/timeseries.min.json',
+    'https://api.covid19india.org/v4/min/timeseries.min.json',
```

Comment:
Can we move `https://api.covid19india.org/v4/min` or `https://api.covid19india.org/v4/` to a constant in `constants.js`,
```js
const BASE_API_URL = https://api.covid19india.org/v4/min;
```
and use it like
```js
`BASE_API_URL/timeseries.min.json`
```
I see the usage of this URL at a lot of places so this will help in **reusing** and **changing** versions or refactoring(in future) easily at one place and will be reflected across every reference of `BASE_API_URL`

Reply:
Sure